File: 3693/3693.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var climbingStairsIi = function(root) {
    if (!root) return 0;
    return 1 + Math.max(climbingStairsIi(root.left), climbingStairsIi(root.right));
};

File: 3693/3693.go
func climbingStairsIi(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := climbingStairsIi(root.Left)
	r := climbingStairsIi(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}